modules/storages/lib/api/v3/project_storages/project_storage_representer.rb

```diff
@@ -47,10 +47,14 @@ class ProjectStorageRepresenter < ::API::Decorators::Single
     end
 
     link :open do
+      next unless can_read_files
```

Comment:
🔴 this is currently not correct, as it would hide this link also for project storages with inactive or manual project folders.

I'd rephrase the helper method to `show_open_storage_links`, and include the check for only `automatic_management_enabled?`